Celigo.ServiceManager.NetSuite.REST/src/RestClient.cs

```diff
@@ -0,0 +1,228 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Reflection;
+using System.Security.Cryptography;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using System.Web;
+using Microsoft.Extensions.Options;
+
+namespace Celigo.ServiceManager.NetSuite.REST
+{
+    public interface IRestClient
```

Comment:
Why are we using `I` herer ? @sameera

Reply:
Since this project is open source, we are being idiomatic :)